### ISSUE TITLE
Optionally output non-sharded data

### ISSUE THREAD

**Issue by Hugos68** [CloudCannon/pagefind#526]
Is there a way to get the same type of results that pagefind.search(...) returns but for all indexed data?
I am trying to create a treeview and need all indexed data to render it.

**Comment by Hugos68** [comment]
Yeah that sounds good, thanks! I don't know if you're one of the contributors but if you are thank you for this amazing piece of tech, it's mind blowing how fast it works without needing infrastructure or services like angolia.

**Comment by bglw** [comment]
I am! So thank you very much for the kind words 😊 

I’ll let you know when this feature lands — it shouldn’t be too difficult, but with the holiday period might take a moment to get to 🙂

**Comment by ColeDCrawford** [comment]
This would be useful! I would also love to be able to start displaying some results without needing to pre-fill a search. Just get the top n results so a user can see what kind of content they are working with. That can be a bit tricky as the default search UI is just the bar, no facets until someone starts typing. Showing the facets by default would be another useful option.
